addons/spreadsheet/static/src/pivot/pivot_model.js

```diff
@@ -13,6 +13,70 @@ import { pivotTimeAdapter } from "./pivot_time_adapters";
 const { toString, toNumber, toBoolean } = helpers;
 const { DEFAULT_LOCALE } = constants;
 
+class PivotMeasure {
+    constructor(metaData, fieldName) {
+        this._metaData = metaData;
+        this._fieldName = fieldName;
+    }
+
+    get display() {
+        return this._fieldName === "__count"
+            ? _t("Count")
+            : this._metaData.fields[this._fieldName].string;
+    }
+
+    get name() {
+        return this._fieldName;
+    }
+}
+
+class PivotDimension {
+    constructor(metaData, dimension) {
+        this._metaData = metaData;
+        const { field, aggregateOperator } = parseGroupField(metaData.fields, dimension);
+        this._field = field;
+        this._aggregateOperator = aggregateOperator;
+        this._fullName = dimension;
+    }
+
+    get display() {
+        return (
+            this._field.string + (this._aggregateOperator ? ` (${this._aggregateOperator})` : "")
+        );
+    }
+
+    get fullName() {
```

Comment:
it's not clear to me what is the full name. Any other name idea for this property ? or at least add a docstring with a example